flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -103,6 +103,26 @@ class TableEnvironmentTest {
     assertEquals(TableTestUtil.replaceStageId(expected), TableTestUtil.replaceStageId(actual))
   }
 
+  @Test
+  def testStreamTableEnvironmentExecutionExplain(): Unit = {
```

Comment:
please also add some test cases for legacy planner

Reply:
Hi @godfreyhe , Sry, How i get legacy planner and add tests for it? it also put in `TableEnvironmentTest`?

Reply:
you can add tests into `org.apache.flink.table.api.stream.ExplainTest` and `org.apache.flink.table.api.batch.ExplainTest`

Reply:
Okay